extra/2-piping.js

```diff
@@ -16,26 +16,29 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  return a + b;
 }
 
-function multiply() {
-
+function multiply(a, b) {
+  return a * b;
 }
 
-function format() {
+function format(digit) {
+  return `£${digit.toString()}`;
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+let badCode = format((startingValue + 10) * 2);  
+
+// functions are not really used, so it would work just for one case
```

Comment:
The badcode is really hard to read and is doing too much on one line so is prone to bugs :)